collector/bluna_token_info.go

```diff
@@ -0,0 +1,81 @@
+package collector
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/lidofinance/terra-monitors/client"
+	"github.com/lidofinance/terra-monitors/client/wasm"
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	BlunaTotalSupply Metric = "bluna_total_supply"
+)
+
+func NewBlunaTokenInfoMintor(address string) BlunaTokenInfoMonitor {
+	return BlunaTokenInfoMonitor{
+		State:           &TokenInfoResponse{},
+		ContractAddress: address,
+	}
+}
+
+type BlunaTokenInfoMonitor struct {
+	State           *TokenInfoResponse
+	ContractAddress string
+	apiClient       *client.TerraLiteForTerra
+	logger          *logrus.Logger
+}
+
+func (h BlunaTokenInfoMonitor) Name() string {
+	return "BlunaTokenInfo"
+}
+
+func (h *BlunaTokenInfoMonitor) Handler(ctx context.Context) error {
+	rewardreq, rewardresp := GetCommonTokenInfoPair()
+	reqRaw, err := json.Marshal(&rewardreq)
+	if err != nil {
+		return fmt.Errorf("failed to marshal BlunaTokenInfo request: %w", err)
+	}
+	p := wasm.GetWasmContractsContractAddressStoreParams{}
+	p.SetContext(ctx)
+	p.SetContractAddress(h.ContractAddress)
+	p.SetQueryMsg(string(reqRaw))
+	resp, err := h.apiClient.Wasm.GetWasmContractsContractAddressStore(&p)
```

Comment:
usually "do — handle error" blocks are joint and visually separated from surrounding code by newlines. could you please visually organize the code in a consistent and common manner? i'm really sorry for carping, this is with peace and love : )

Reply:
done, i was trying to organize code better.